test/apartment_test.rb

```diff
@@ -29,8 +29,9 @@ def test_that_add_renter_returns_and_instance_of_renter
     joe = Renter.new("Joe")
 
     a1.add_renter(joe)
-    
+
     assert_instance_of Renter, a1.renter
+    assert_equal joe, a1.renter
```

Comment:
Since we made the renter before we passed it in, we can assert that its the same one coming back when we call it.